docs/troubleshooting.md

```diff
@@ -0,0 +1,12 @@
+# Tests fail on 'Suite Setup' - 'To Boot Directly' not found
+
+If DUT is configured to use PiKVM it is important to make sure that serial
```

Comment:
The actual problem is the default settings of Serial Redirection option in firmware. On some platforms (laptops, MSI, it is disabled by default). It may be worth mentioning, that this is not directly cause by the PiKVM itself, but it just happens that if we use PiKVM, we typically disable serial by default, so users can have shorter boot time by default, and only enable serial console if really needed.